opensearch_dashboards.json

```diff
@@ -17,7 +17,8 @@
   ],
   "optionalPlugins": [
     "dataSource",
-    "dataSourceManagement"
+    "dataSourceManagement",
+    "usageCollection"
```

Comment:
Not sure telemetry plugin is needed, @chishui

Reply:
Guess we don't? Didn't see where `telemetry` is used.

Reply:
Oh, I think we don't need `telemetry`